account/models.py

```diff
@@ -30,19 +20,21 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     )
     email = models.EmailField(_('email address'), unique=True)
-    phone = models.CharField(_('phone number'), unique=True, max_length=15)
+    phone = models.FloatField(_('phone number'), unique=True, max_length=11)
     first_name = models.CharField(_('first name'), max_length=30, blank=True)
     last_name = models.CharField(_('last name'), max_length=30, blank=True)
-    account_num = models.FloatField(_('account num'),  blank=True, unique=True)
+    account_num = models.FloatField(_('account num'), blank=True, unique=True, default=0)
```

Comment:
I believe you can use the IntergerField instead for account number
https://docs.djangoproject.com/en/4.0/ref/models/fields/#integerfield